driver-core/src/main/java/com/datastax/driver/core/SimpleStatement.java

```diff
@@ -77,16 +77,13 @@ public SimpleStatement(String query) {
      *
      * @param query the query string.
      * @param values values required for the execution of {@code query}.
-     *
-     * @throws IllegalArgumentException if one of {@code values} is not of a type
-     * corresponding to a CQL3 type, i.e. is not a Class that could be returned
-     * by {@link DataType#asJavaClass}.
+     * @throws IllegalArgumentException if the number of values is greater than 65535.
      */
     public SimpleStatement(String query, Object... values) {
         if (values.length > 65535)
             throw new IllegalArgumentException("Too many values, the maximum allowed is 65535");
         this.query = query;
-        this.values = convert(values);
+        this.values = values;
```

Comment:
It could be useful to save data types also in addition to `values`. If I understand well, here `getValues()` or `getObject()` will return `Object`s but we cannot know how to use this object without the data type, right ?